ecosystem/ecosystem.html

```diff
@@ -31,10 +31,9 @@ <h1>
 
       <div id="ecosystem-index-cards">
         <div class="list row ecosystem-cards-wrapper cards-right">
-          {% assign ecosystem = site.ecosystem  | sort_natural: "title" %}
-
+          {% assign ecosystem = site.ecosystem | sample: 100 %}
```

Comment:
Is there a way to pass in the array size as the sample? That way if they ever go over 100 cards it'll still work.

Reply:
If they want the random sort to happen on each page load, it might be possible to use the existing list.js library. https://listjs.com/api/#sort The `sort` function has a `sortFunction` argument. Maybe you can use that argument to pass in a function that randomizes the cards.

Reply:
> If they want the random sort to happen on each page load, it might be possible to use the existing list.js library. https://listjs.com/api/#sort The sort function has a sortFunction argument. Maybe you can use that argument to pass in a function that randomizes the cards.

After the meeting last Friday, Andres said the current implementation is fine, so you can ignore this comment.